packaging/installer/methods/manual.md

```diff
@@ -201,11 +199,43 @@ cd netdata
 
-Once the installer completes, the file `/etc/netdata/netdata.conf` will be created (if you changed the installation directory, the configuration will appear in that directory too).
-
-You can edit this file to set options. One common option to tweak is `history`, which controls the size of the memory database Netdata will use. By default is `3600` seconds (an hour of data at the charts) which makes Netdata use about 10-15MB of RAM (depending on the number of charts detected on your system). Check **\[[Memory Requirements]]**.
-
-To apply the changes you made, you have to restart Netdata.
+## Optional parameters to alter your installation
+
+`netdata-installer.sh` accepts a few parameters to customize your installation:
+
+-   `--dont-wait`: Enable automated installs by not prompting for permission to install any required packages.
+-   `--dont-start-it`: Prevent the installer from starting Netdata automatically.
+-   `--stable-channel`: Automatically update only on the release of new major versions.
+-   `--nightly-channel`: Automatically update on every new nightly build.
+-   `--disable-telemetry`: Opt-out of [anonymous statistics](/docs/anonymous-statistics.md) we use to make
+    Netdata better.
+-   `--no-updates`: Prevent automatic updates of any kind.
+-   `--reinstall`: If an existing install is detected, reinstall instead of trying to update it. Note that this
+    cannot be used to change installation types.
+-   `--local-files`: Used for [offline installations](offline.md). Pass four file paths: the Netdata
+    tarball, the checksum file, the go.d plugin tarball, and the go.d plugin config tarball, to force kickstart run the
+    process using those files. This option conflicts with the `--stable-channel` option. If you set this _and_
+    `--stable-channel`, Netdata will use the local files.
+
+### Claim your node during installation
+
+The `netdata-installer.sh` script accepts additional parameters to automatically [claim](/claim/README.md) your node to
+Netdata Cloud immediately after installation. Find the `token` and `rooms` strings by [signing in to Netdata
+Cloud](https://app.netdata.cloud/sign-in?cloudRoute=/spaces), then clicking on **Claim Nodes** in the [Spaces management
+area](https://learn.netdata.cloud/docs/cloud/spaces#manage-spaces).
+
+- `--claim-token`: The unique token associated with your Space in Netdata Cloud.
+- `--claim-rooms`: A comma-separated list of tokens for each War Room this node should appear in.
+- `--claim-proxy`: Should take the form of `socks5[h]://[user:pass@]host:ip` for a SOCKS5 proxy, or
+  `http://[user:pass@]host:ip` for an HTTP(S) proxy.See [claiming through a
+  proxy](/claim/README.md#claim-through-a-proxy) for details.
+- `--claim-url`: Defaults to `https://app.netdata.cloud`.
+
+For example:
+
+```bash
+./netdata-installer.sh --claim-token=TOKEN --claim-rooms=ROOM1,ROOM2
+```
```

Comment:
We actually don’t support auto-claiming when installing directly using `netdata-installer.sh`. The original implementation did, but some of the required changes resulted in the associated code needing to be in the kickstart scripts instead of the installer script.

That said, adding something here about post-install claiming using the `netdata-claim.sh` script would probably be a good idea.